config/default.yml

```diff
@@ -990,6 +990,11 @@ Style/LambdaCall:
     - braces
 
 Style/MethodCallWithArgsParentheses:
+  Exclude:
```

Comment:
@mikegee: Yeah ... after sleeping on it, I'm starting to lean towards disabled by default again. 😞